src/solver/opamCudf.ml

```diff
@@ -850,8 +838,8 @@ let extract_explanations packages cudfnv2opam reasons : explanation list =
   in
   let print_set pkgs =
```

Comment:
It is no more `print_set`, but something like `construct_set` ?

Reply:
fixed in 3b9d8032e